aep/general/0148/aep.md.j2

```diff
@@ -1,5 +1,155 @@
+name. Resources **must not** use `last_name` for this concept, because the
+family name is not placed last in many cultures.
+
+### Timestamps
+
+See AEP-213 for more information about common components for `Timestamp` and
+`Duration`.
+
+#### create_time
+
+The output only `Timestamp create_time` field **must** represent the timestamp
+when the resource was created. This **may** be either the time creation was
+initiated or the time it was completed. [Declarative-friendly resources][]
+**should** include this field.
+
+#### update_time
+
+The output only `Timestamp update_time` field **must** represent the timestamp
+when the resource was most recently updated. Any change to the resource made by
+users **must** refresh this value; changes to a resource made internally by the
+service **may** refresh this value. [Declarative-friendly resources][]
+**should** include this field.
+
+#### delete_time
+
+The output only `Timestamp delete_time` field **must** represent the timestamp
+that a resource was soft deleted. This **may** correspond to either the time
+when the user requested deletion, or when the service successfully soft deleted
+the resource. If a resource is not soft deleted, the `delete_time` field
+**must** be empty.
+
+Resources that support soft delete (AEP-164) **should** provide this field.
+
+#### expire_time
+
+The `Timestamp expire_time` field **should** represent the time that a given
+resource or resource attribute is no longer useful or valid (e.g. a rotating
+security key). It **may** be used for similar forms of expiration as described
+in AEP-214.
+
+Services **may** provide an `expire_time` value that is inexact, but the
+resource **must not** expire before that time.
+
+#### purge_time
+
+The `Timestamp purge_time` field **should** represent the time when a soft
+deleted resource will be purged from the system (see AEP-164). It **may** be
+used for similar forms of expiration as described in AEP-214. Resources that
+support soft delete **should** include this field.
+
+Services **may** provide a `purge_time` value that is inexact, but the resource
+**must not** be purged from the system before that time.
+
+### Well known string fields
+
+#### IP address
+
+A field that represents an IP address **must** comply with the following:
+
+- use type `string`
+- use the name `ip_address` or end with the suffix `_ip_address` e.g.
+  `resolved_ip_address`
+- specify the IP address version format via one of the supported formats
+  `IPV4`, `IPV6`, or if it can be either, `IPV4_OR_IPV6` (see AEP-202)
+
+#### uid
+
+The output only `string uid` field refers to a system-assigned unique
+identifier for a resource. When provided, this field **must** be a [UUID4][]
+and **must** specify this format via the `UUID4` format extension (see
+AEP-202). [Declarative-friendly resources][] **should** include this field.
+
+## Further reading
+
+- For standardized codes, see AEP-143.
```

Comment:
I think we should use the slug names here. Or maybe we can just file a cleanup ticket for the whole repository at some point?

Reply:
Yeah, I think a batch cleanup would work best, but also updated this instance.

Reply:
I could go either way on whether we add it to the site-generator - I don't feel like it improves things much and it's not handled natively in markdown files.

But sounds good on a batch-fix (although I prefer we fix things along the way so the batch is smaller)